apps/app/components/Profile/index.tsx

```diff
@@ -40,6 +40,7 @@ function Profile({ id, role }: Props) {
   const [badges, setBadges] = useState<any[]>([]);
   const [projects, setProjects] = useState<any[]>([]);
   const [skills, setSkills] = useState<any[]>([]);
+  const [date, setDate] = useState<any>(String);
```

Comment:
```suggestion
  const [date, setDate] = useState<String>("");
```

I think you want this